assets/ajax-cart.js.liquid

```diff
@@ -375,6 +408,7 @@ var ajaxCart = (function(module, $) {
 
       // If it has a data-line, update the cart
       if (line) {
+        $body.trigger('adjustCart.ajaxCart', [this, line, qty]);
```

Comment:
The entire cart reprints, so having the element available won't do much help. Open to a rebuttal though.